docs/howtos/eviction-placement-disruption-budget.md

```diff
@@ -0,0 +1,356 @@
+  - lastTransitionTime: "2025-01-19T12:36:54Z"
+    message: No override rules are configured for the selected resources
+    observedGeneration: 2
+    reason: NoOverrideSpecified
+    status: "True"
+    type: ClusterResourcePlacementOverridden
+  - lastTransitionTime: "2025-01-19T12:36:54Z"
+    message: Works(s) are succcesfully created or updated in 1 target cluster(s)'
+      namespaces
+    observedGeneration: 2
+    reason: WorkSynchronized
+    status: "True"
+    type: ClusterResourcePlacementWorkSynchronized
+  - lastTransitionTime: "2025-01-19T12:36:54Z"
+    message: The selected resources are successfully applied to 1 cluster(s)
+    observedGeneration: 2
+    reason: ApplySucceeded
+    status: "True"
+    type: ClusterResourcePlacementApplied
+  - lastTransitionTime: "2025-01-19T12:36:54Z"
+    message: The selected resources in 1 cluster(s) are available now
+    observedGeneration: 2
+    reason: ResourceAvailable
+    status: "True"
+    type: ClusterResourcePlacementAvailable
+  observedResourceIndex: "0"
+  placementStatuses:
+  - clusterName: kind-cluster-1
+    conditions:
+    - lastTransitionTime: "2025-01-19T12:36:54Z"
+      message: 'Successfully scheduled resources for placement in "kind-cluster-1"
+        (affinity score: 0, topology spread score: 0): picked by scheduling policy'
+      observedGeneration: 2
+      reason: Scheduled
+      status: "True"
+      type: Scheduled
+    - lastTransitionTime: "2025-01-19T12:36:54Z"
+      message: Detected the new changes on the resources and started the rollout process
+      observedGeneration: 2
+      reason: RolloutStarted
+      status: "True"
+      type: RolloutStarted
+    - lastTransitionTime: "2025-01-19T12:36:54Z"
+      message: No override rules are configured for the selected resources
+      observedGeneration: 2
+      reason: NoOverrideSpecified
+      status: "True"
+      type: Overridden
+    - lastTransitionTime: "2025-01-19T12:36:54Z"
+      message: All of the works are synchronized to the latest
+      observedGeneration: 2
+      reason: AllWorkSynced
+      status: "True"
+      type: WorkSynchronized
+    - lastTransitionTime: "2025-01-19T12:36:54Z"
+      message: All corresponding work objects are applied
+      observedGeneration: 2
+      reason: AllWorkHaveBeenApplied
+      status: "True"
+      type: Applied
+    - lastTransitionTime: "2025-01-19T12:36:54Z"
+      message: All corresponding work objects are available
+      observedGeneration: 2
+      reason: AllWorkAreAvailable
+      status: "True"
+      type: Available
+  selectedResources:
+  - kind: Namespace
+    name: test-ns
+    version: v1
+```
+> **Note:** The `ClusterResourcePlacementDisruptionBudget` object is only used as an information source by the eviction controller and hence it doesn't have a status associated to it.
+
+Now we will create a `ClusterResourcePlacementDisruptionBudget` object to protect resources on the member cluster from voluntary disruption:
```

Comment:
Hi Arvind! Some explanation on how this DB can protect might be helpful here I think.

Reply:
(Or in other words, why won't the eviction below be processed by Fleet)